libs/ha/ha_common_libs_k8s.py

```diff
@@ -1498,6 +1506,9 @@ def delete_kpod_with_shutdown_methods(self, master_node_obj, health_obj,
                 pod_info[pod]['deployment_name'] = resp[2]
             pod_info[pod]['method'] = down_method
             pod_info[pod]['hostname'] = hostname
+            if event is not None:
```

Comment:
Or condition can be if event.is_set()

Reply:
> Or condition can be if event.is_set()

we can not set this. as in we have kept event = None, and in non-event it will fail with error None type can't have is_set() someting.